### DISCUSSION TITLE
Can someone provide (.dwg) files that does not have errors (Unity related)

### DISCUSSION THREAD

**Discussion by JamesMorDigi** [iTwin/itwinjs-core#4192] [General]
I'm following this tutorial https://www.youtube.com/watch?v=Zb5GecN-2Lg&t=809s&ab_channel=BentleyiTwinPlatform

But when I upload my own model, that model does not display in Unity
![image](https://user-images.githubusercontent.com/47785665/187482102-73e6ccae-df2c-444b-9bf0-1b2474c5ed32.png)

When I use the Bentley sample models, the unity project works fine

I have followed all the steps correctly as specified https://www.itwinjs.org/learning/tutorials/create-test-imodel-itwin-sync/#create-imodel

but I keep getting this error
![image](https://user-images.githubusercontent.com/47785665/187480182-e87615ca-c796-4fdb-a81b-721b11eef9c6.png)

Also, where can I get a copy of the sample files from Irwin/Bently Sample? e.g Stadium, Metro station etc

**Comment by tamanud** [comment]
@mgooding can you please help here

**Reply by mgooding** [reply]
Sorry I missed this when it was posted. These look like two separate issues:

1) **Your iModel does not display in Unity** - imodel-unity-example only loads the geometry that's within the camera frustum. If your iModel contains Saved Views, you'll automatically start at the first one. Based on your screenshot, I don't think your iModel contains any saved views, so you've started at an arbitrary position that might not have an geometry. If you move your camera around, you should see geometry start to load.

2) **Synchronization failure** - @jffmarker should be able to help here but will need more information.

As for your other question:
> Also, where can I get a copy of the sample files from Irwin/Bently Sample? e.g Stadium, Metro station etc

You can follow the steps here: https://developer.bentley.com/tutorials/web-application-quick-start/#optional-manual-imodel-creation-process
You can then copy the iTwin and iModel ID as described in that tutorial for use in imodel-unity-example or any of our other samples.

**Reply by jffmarker** [reply]
@tamanud or @mgooding, do you know if there's a way @JamesMorDigi could provide the cloud sync run ID so we can dig up the logs? Otherwise, I think we'll need the source data to try to reproduce anything.

**Reply by jffmarker** [reply]
@donhfu for DWG-related issues as well.

**Reply by donhfu** [reply]
@JamesMorDigi , your link of create imodel had a step "Download iTwin synchronizer".  The link "iTwin Synchronizer" appears to lead to a different page.  That might be where your problem had started.  You may follow this link to learn about iTwin Synchronizer:

https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/47597/itwin-synchronizer-client

Basically, you need to run ITSYNC on your local computer where you have your source DWG files.  If your DWG files were created from AutoCAD, you may select AutoCAD connector.  It will create an iModel from a DWG file you have selected.  Once you have an iModel, then you may use Unity or other iTwin services.
